upgrading/automated_upgrades.adoc

```diff
@@ -756,6 +797,34 @@ You can set the `openshift_logging_elasticsearch_replace_configmap` parameter to
 ConfigMap can cause the upgrade to fail. The default is set to `false`. For more information, see the parameter 
 in xref:../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[specify logging Ansible variables].
 
+. Scale down your Fluentd and Elasticsearch pods to stop data ingestion and ensure the cluster state does not change.
+
+. Perform an link:https://www.elastic.co/guide/en/elasticsearch/reference/2.4/indices-flush.html[Elasticsearch Index flush] on all relevant indices. The flush process persists all logs from memory to disk, which prevents log loss when Elasticsearch is shutdown during the upgrade.
+
+. Use the link:https://www.elastic.co/guide/en/elasticsearch/reference/5.5/modules-snapshots.html[Elasticsearch snapshot module] to create a snapshot of your AWS volumes.
+
+. Run the following script to determine if your indices contain any fields with a dot in the name.
```

Comment:
Which scripts?  Could you tell me how to open the whole document? I  can only view the diff part?

Reply:
@anpingli For full preview, see:
http://file.rdu.redhat.com/~mburke/logging-311-upgrade/upgrading/automated_upgrades.html#upgrading-efk-logging-stack

Reply:
@mburke5678 Thanks, I found the script at http://file.rdu.redhat.com/~mburke/logging-311-upgrade/upgrading/automated_upgrades.html#upgrading-efk-logging-stack